content/docs/getting-started/introduction.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Introductory Tutorial
+title: Introduction
 id: introduction
 prev: null
 next: /docs/getting-started/cms-set-up
+last_edited: '2020-08-06T16:09:22.451Z'
 ---
+Howdy, Welcome to Tina! We are so happy you landed on this piece of heaven on the internet. The goal of this tutorial is to guide you through setting up and deploying your first site with TinaCMS using a `create-react-app`  starter template. This tutorial will walk you through Tina's fundamental building blocks by setting up editing on a basic `create-react-app` [demo](https://github.com/tinacms/tina-intro-tutorial).
```

Comment:
```suggestion
Howdy, Welcome to Tina! This tutorial will walk you through Tina's fundamental building blocks by setting up editing on a basic `create-react-app` [demo](https://github.com/tinacms/tina-intro-tutorial).
```

Reply:
We don't cover deployment. The initial 'goal of this tutorial' feels redundant with the follow up sentence. Also, we want to avoid the word 'starter' when referring to the demo.